### ISSUE TITLE
poly_between doesn't exist anymore in matplotlib 3.1

### ISSUE THREAD

**Issue by scls19fr** [python-windrose/windrose#81]
From #77 

```python
ax = WindroseAxes.from_ax()
ax.contourf(wd, ws, bins=np.arange(0, 8, 1), cmap=cm.hot)
ax.set_legend()
```

displays the following warning

```
 MatplotlibDeprecationWarning: The poly_between function was deprecated in version 2.2. xs, ys = poly_between(angles, 0, val)
```

https://github.com/python-windrose/windrose/blob/e816aa8beb9945637d1b77366a199f321dac2bad/windrose/windrose.py#L507-L515

**Comment by yoonghm** [comment]
These functions may not be used internally in Matplotlib but they are used elsewhere. Please keep them.

**Comment by scls19fr** [comment]
A workaround is given at https://github.com/matplotlib/matplotlib/issues/11911#issuecomment-414982843

**Comment by akrherz** [comment]
Note that `poly_between` is now gone with the newly released matplotlib 3.1

**Comment by schmidt-uni-leipzig** [comment]
+1

Please fix this issue, because with matplotlib 3.1 one gets the following error

`ImportError: cannot import name 'poly_between' from 'pylab'`

**Comment by scls19fr** [comment]
I'm currently away... with only a phone

Could you provide a PR?

A least could you try the workaround described previously?
